test/src/network.spec.ts

```diff
@@ -853,14 +853,19 @@ describe('network', function () {
         await page.goto(httpsServer.PREFIX + '/setcookie.html');
 
         const response = await new Promise<HTTPResponse>(resolve => {
-          page.on('response', resolve);
+          page.on('response', async response => {
+            if ((await response.text()) === '') {
```

Comment:
why do we await the response with an empty text? could we leave a comment which response it is or perhaps use the URL for matching?